scr/hw10/FileNavigator.java

```diff
@@ -0,0 +1,80 @@
+package hw10;
+
+import java.util.*;
+
+public class FileNavigator {
+    private Map<String, List<FileData>> fileStorage;
+
+    public FileNavigator() {
+        fileStorage = new HashMap<>();
+    }
+
+    public void add(String filePath, FileData fileInfo) {
+        if (!Objects.equals(filePath, fileInfo.getFilePath())) {
+            System.out.printf("paths not equals ", filePath);
+        } else {
+            if (fileStorage.containsKey(filePath)) {
+                fileStorage.get(filePath).add(fileInfo);
+            } else {
+                List<FileData> section = new ArrayList<>();
+                section.add(fileInfo);
+                fileStorage.put(filePath, section);
+            }
+        }
+    }
+
+    public List<String> find(String path) {
+        List<String> namesToReturn = new ArrayList<>();
+
+        if (fileStorage.containsKey(path)) {
+            for (FileData elem : fileStorage.get(path)) {
+                namesToReturn.add(elem.getFileName());
+            }
+        } else {
+            System.out.printf("File path is not found ", path);
+        }
+        return namesToReturn;
```

Comment:
can be simplified to:
`return fileStorage.containsKey(path) ? fileStorage.get(path) : emptyList();`